### DISCUSSION TITLE
change localization Format

### DISCUSSION THREAD

**Discussion by enghussein** [abpframework/abp#10514] [Q&A]
currently, i'm using ABP Version="5.0.0-beta.2" 
my question what is the best way to change the default format of the localization language especially for Arabic, I want to set NumberFormat.NumberDecimalSeparator = "."  instead of ","

**Comment by maliming** [comment]
hi

I think you can create a new middleware and try to change `CultureInfo.CurrentCulture` and `CultureInfo.CurrentUICulture` after `UseAbpRequestLocalization`.
